node_modules/oae-core/tenantlandingpage/css/tenantlandingpage.css

```diff
@@ -0,0 +1,154 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#tenantlandingpage {
+    font-weight: 200;
+    margin: 0 auto;
+    max-width: 985px;
```

Comment:
There's some padding missing around the widget that's causing its content to start to high up and be slightly too wide. When comparing the old landing page against the new one, they should be exactly the same.